remix-gql-tailwind/vitest.config.ts

```diff
@@ -0,0 +1,15 @@
+/// <reference types="vitest" />
+/// <reference types="vite/client" />
+
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import tsconfigPaths from "vite-tsconfig-paths";
+
+export default defineConfig({
+  plugins: [react(), tsconfigPaths()],
+  test: {
+    globals: true,
+    environment: "happy-dom",
```

Comment:
just a note I tried using happy-dom once and had some issues with some of the event apis from testing library if you ever notice an issue there

Reply:
Good to know. Making a note of this.